matrimony_admin/forms.py

```
@@ -1,10 +1,11 @@
 from typing import Any
 from django import forms
 from U_messages.models import NotificationDetails,AmidUsers
-from U_auth.models import costume_user
+from U_auth.models import costume_user, UserPersonalDetails
 
 # import get_object_or_404()
 from django.shortcuts import get_object_or_404
+from U_auth.models import UserPersonalDetails
```

Ruff

Redefinition of unused `UserPersonalDetails` from line 4

Remove definition: `UserPersonalDetails`

(F811)